app/uk/gov/hmrc/cdsreimbursementclaim/models/claim/CompleteClaimAnswer.scala

```diff
@@ -16,12 +16,11 @@
 
 package uk.gov.hmrc.cdsreimbursementclaim.models.claim
 
-import play.api.libs.json.{Json, OFormat}
+import cats.data.NonEmptyList
 
-final case class CompleteClaimAnswer(
-  claims: List[Claim]
-)
+object ClaimsAnswer {
```

Comment:
@gabor-dorcsinecz please can you move it to `answers` package

Reply:
Done